front-end-notes/jQuery-and-lodash.md

```diff
@@ -0,0 +1,22 @@
+#jQuery and lodash#
+
+Whereas jQuery is the Swiss Army knife of DOM, Lodash is the equivalent of the Batman’s utility belt for Javascript. And just like Batman who always has some gadgets in his trusty belt to get out of sticky situation, Lodash comes with a lot of goodies.
+
+##jQuery##
+
+jQuery is a fast, concise, JavaScript Library that simplifies how you traverse HTML documents, handle events, perform animations, and add Ajax interactions to your web pages. jQuery is designed to change the way that you write JavaScript. http://jquery.com/
+
+- Cheat Sheet: http://oscarotero.com/jquery/
+- http://teamtreehouse.com/library/q:jquery
+- https://www.codeschool.com/paths/javascript
+- https://github.com/jquery/learn.jquery.com/blob/master/page/about-jquery/how-jquery-works.md
+
+
+##lodash##
+
+A JavaScript utility library delivering consistency, modularity, performance, & extras.
+
+- https://github.com/lodash/lodash
+- https://lodash.com/
+- http://colintoh.com/blog/lodash-10-javascript-utility-functions-stop-rewriting
+

```

Comment:
Seth, we left out the jQuery docs since they are linked from the cheat sheet, but we can add them in if other want them